### ISSUE TITLE
image tag float left and right is not supported

### ISSUE THREAD

**Issue by puahhhh** [daohoangson/flutter_widget_from_html#1001]
## Use case
Image tag float left and right is not supported in this package, is there any work around to ensure that the image can be float left or right 

Here is a sample result.

<img width="856" alt="Screenshot 2023-08-23 at 4 45 11 PM" src="https://github.com/daohoangson/flutter_widget_from_html/assets/130420407/97d0fde9-b55e-428b-b2ce-07cd5fbf2562">

**Comment by puahhhh** [comment]
@daohoangson HELPPPPPPPP 😭

**Comment by daohoangson** [comment]
I don't think we can support CSS `float` with the existing text painting system in Flutter. A package probably has to layout text line by line to achieve this effect.

**Comment by puahhhh** [comment]
in the end, we used both solution to display the html content by using "flutter_widget_from_html_core" and "html_editor_enhanced" package to display but with some restriction for each package used. Hope to get a solution for html "float" property soon in flutter packages

**Comment by banknanthawat** [comment]
Can the float_column library be used to support float styles?

**Comment by banknanthawat** [comment]
> I don't think we can support CSS `float` with the existing text painting system in Flutter. A package probably has to layout text line by line to achieve this effect.

Can the float_column library be used to support float styles?

**Comment by daohoangson** [comment]
https://pub.dev/packages/float_column does look promising.

**Comment by banknanthawat** [comment]
> https://pub.dev/packages/float_column does look promising.

If you have time, I would like you to create an example using float_column to solve this issue so that people facing this problem will have an option to resolve the float style issue.
